packages/hmr-plugin/src/symbols.ts

```diff
@@ -1,6 +1,10 @@
 import { StateContext } from '@ngxs/store';
 import { NgModuleRef } from '@angular/core';
 
+export enum HMR_RUNTIME {
```

Comment:
Should be `const enum`? Also `HrmRuntime`, TS style guide says:
> Use PascalCase for enum names

Reply:
done